egov/egov-collection/src/main/java/org/egov/collection/service/RemittanceServiceImpl.java

```diff
@@ -787,9 +787,8 @@ private void groupByReceiptDate(Map<String, List<Receipt>> receiptDateWiseMap, L
      *
      * @return List of HashMap
      */
-    @Override
```

Comment:
Add annotation back and change method signature in the interface instead of removing override annotation